### PR TITLE
Partner referrals

### PR DESCRIPTION
**Pull request by barbaraliau**
Partner model to keep track of referrals and rev share

Connected with: https://github.com/Storj/bridge/pull/450

### CODE REVIEW

File: lib/models/partner.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    index: true
+  },
+  /*
+   * Percentage is in decimal form, range from 0.0 to 1.0
+   */
+  revSharePercentage: {
+    type: Number,
+    min: 0,
+    max: 1,
+    required: true
```
Comment (pgerbes1) [inline lib/models/partner.js]: Might not want this to be required as it's possible future deals will use a different metric (e.g. $ per gb downloaded).

Comment (barbaraliau) [reply lib/models/partner.js]: Ah. Makes sense. Do we want a default of `0`?

Comment (pgerbes1) [reply lib/models/partner.js]: Sure I think that makes sense

**Comment by barbaraliau** [comment]
Ready for review @braydonf.

**Comment by pgerbes1** [comment]
This looks good to me. I'll hold off on a merge until @braydonf has had a chance to peruse.

**Comment by braydonf** [comment]
LGTM. Testing creating new user locally.

**Pull request merged**